packages/integrations/netlify/src/integration-functions.ts

```diff
@@ -85,6 +86,16 @@ function netlifyFunctions({
 				}
 			},
 			'astro:build:done': async ({ routes, dir }) => {
+				const functionsConfig = {
+						version: 1,
+						config: {
+							nodeModuleFormat: "esm"
+						}
+				}
+				const functionsConfigPath = join(fileURLToPath(_config.build.server), "entry.json")
+				await writeFile(functionsConfigPath, JSON.stringify(functionsConfig))
+
+				// await writeFile(_config.build.server)
```

Comment:
To remove?

Reply:
Yes, to remove! Not sure what happened there, should get some coffee :D